### PR TITLE
accounting spec update

### PR DESCRIPTION
**Pull request by enkore**
Also moved & separated the accounting APIs to an extra page.

### CODE REVIEW

File: _docs/Qabel-Accounting.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+---
+title: "Account management"
+---
+# Account management
+
+## Abstract
+
+The accounting server keeps track of registered users (=accounts !=
+identities) and authorizes them to other services by creating
+temporary authorization tokens.
```
Comment (julianseeger) [inline _docs/Qabel-Accounting.md]: currently not temporary afaik

File: _docs/Qabel-Protocol-Box.md [hunk 2]
```diff
@@ -9,77 +9,31 @@ A set of protocols to store files and folders on a VOLUME that is (currently) st
...
+Block server enforces the permissions that are connected to the
+authentication tokens.  Qabel Box uses the Block server to store the
+blocks and metadata.
 
-## Accounting server
+## Block server
 
-The accounting server controls write access to the Block server. Registered users ( = accounts != identities) can request temporary authentication tokens for the access.
-All data is sent as JSON and UTF-8. All data types are defined [here](../Qabel-Client-Local-Data#data-types).
+When the blocks server receives a request that requires authorization
```
Comment (julianseeger) [inline _docs/Qabel-Protocol-Box.md]: `'s/blocks/block'`

**Comment by julianseeger** [comment]
didn't check the complete api details but look good :+1:

File: _docs/Qabel-Accounting.md (hunk 1 above)
Comment (enkore) [inline _docs/Qabel-Accounting.md]: Ah, I must've been thinking about the auth token *cache* on the block server

**Comment by roeslpa** [comment]
Looks good to me as well :+1:

**Pull request merged**